src/utilities/api/types.ts

```diff
@@ -29,6 +29,10 @@ export type GiftCardDetails = {
   created_at: string;
   email: string;
   expiration: string | null;
+  gift_card_id: string;
+  // The latest gift card amount's created_at date. This denotes the last
```

Comment:
nit: you can make this a tsdoc comment with /** .. */, so it'll appear for this variable on hover everywhere :)

Reply:
oh nice. done